plugins/mentions/plugin.js

```diff
@@ -385,7 +361,7 @@
 	 * When using asynchronous method i.e. backend URL string or a function,
 	 *  you should provide correct object structure containing unique item ID and a name.
 	 *
-	 * You can pass additional information also which can be used to create custom view template.
+	 * You can pass additional information also which can be used to create custom item template.
```

Comment:
Oh my, how did this fragment get here? :D It's `mentions.config.feed` docs and it's talking about template 🙂 Either it wasn't reviewed in the mention feature issue, or it has slipped by.

Let's fix it here, this information needs to go to a proper place.